### PR TITLE
zh: localize /reference/config/analysis/ist0155/

### PR DESCRIPTION
**Pull request by jzhupup**
Added the zh text to this topic:

`content/zh/docs/reference/config/analysis/ist0155/index.md`

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/config/analysis/ist0155/index.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+title: EnvoyFilterUsesRelativeOperationWithProxyVersion
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `EnvoyFilter` 没有设置优先级并使用相对的补丁操作（`INSERT_BEFORE/AFTER`、`REPLACE`、`MERGE`、`DELETE`）和 `proxyVersion` 时会出现此消息，这可能会导致 `EnvoyFilter` 升级期时没有被应用。使用 `INSERT_FIRST` 或 `ADD` 选项或设置优先级可能有助于确保 `EnvoyFilter` 被正确应用。”关注 `proxyVersion` 的原因是，在升级后，`proxyVersion` 可能会发生变化，并且现在使用它的顺序将与以往不同。
```
Comment (litong01) [inline content/zh/docs/reference/config/analysis/ist0155/index.md:8]: I think that here the term `relative patch` probably should be translated into `相关补丁` not `相对的补丁`, the following maybe a bit better.

没有设置优先级并且没有使用相关补丁操作

The last statement, could be a bit better like this.

在升级后，`proxyVersion` 可能会发生变化，升级后的使用它顺序可能不同与升级前的顺序。

File: content/zh/docs/reference/config/analysis/ist0155/index.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                [":authority"] = "internal.org.net"
+               },
+              "authorize call",
+              5000)
+            end
+{{< /text >}}
+
+## 如何解决
+
+因为 `REPLACE` 的相对操作是与 `proxyVersion` 一起使用，所以添加 `priority` 可以解决这个问题：
```
Comment (litong01) [inline content/zh/docs/reference/config/analysis/ist0155/index.md:94]: 相关操作

**Review by litong01 — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/config/analysis/ist0155/index.md (hunk 1 above)
Comment (jzhupup) [reply content/zh/docs/reference/config/analysis/ist0155/index.md]: Thanks！

Comment (zirain) [reply content/zh/docs/reference/config/analysis/ist0155/index.md]: 不同于

File: content/zh/docs/reference/config/analysis/ist0155/index.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
+---
+title: EnvoyFilterUsesRelativeOperationWithProxyVersion
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `EnvoyFilter` 没有设置优先级并没有使用相关补丁操作（`INSERT_BEFORE/AFTER`、`REPLACE`、`MERGE`、`DELETE`）和 `proxyVersion` 时会出现此消息，这可能会导致 `EnvoyFilter` 升级期时没有被应用。使用 `INSERT_FIRST` 或 `ADD` 选项或设置优先级可能有助于确保 `EnvoyFilter` 被正确应用。关注 `proxyVersion` 的原因是，在升级后，`proxyVersion` 可能会发生变化，升级后的使用它顺序可能不同与升级前的顺序。
```
Comment (zirain) [inline content/zh/docs/reference/config/analysis/ist0155/index.md:8]:
```suggestion
当 `EnvoyFilter` 没有设置优先级并没有使用相关补丁操作（`INSERT_BEFORE/AFTER`、`REPLACE`、`MERGE`、`DELETE`）和 `proxyVersion` 时会出现此消息，这可能会导致 `EnvoyFilter` 升级期时没有被应用。使用 `INSERT_FIRST` 或 `ADD` 选项或设置优先级可能有助于确保 `EnvoyFilter` 被正确应用。关注 `proxyVersion` 的原因是，在升级后，`proxyVersion` 可能会发生变化，升级后它的使用顺序可能不同于升级前的顺序。
```

Comment (zirain) [reply content/zh/docs/reference/config/analysis/ist0155/index.md:8]: `升级后它的使用顺序可能不同于升级前`  这样读起来好像更顺畅

Comment (litong01) [reply content/zh/docs/reference/config/analysis/ist0155/index.md:8]: agreed.

File: content/zh/docs/reference/config/analysis/ist0155/index.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
+---
+title: EnvoyFilterUsesRelativeOperationWithProxyVersion
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `EnvoyFilter` 没有设置优先级并没有使用相关补丁操作（`INSERT_BEFORE/AFTER`、`REPLACE`、`MERGE`、`DELETE`）和 `proxyVersion` 时会出现此消息，这可能会导致 `EnvoyFilter` 升级期时没有被应用。使用 `INSERT_FIRST` 或 `ADD` 选项或设置优先级可能有助于确保 `EnvoyFilter` 被正确应用。关注 `proxyVersion` 的原因是，在升级后，`proxyVersion` 可能会发生变化，升级后它的使用顺序可能不同于升级前的顺序。
```
Comment (litong01) [inline content/zh/docs/reference/config/analysis/ist0155/index.md:8]: can you also please change 升级期时 to 升级时? I am not really sure we ever say things like 升级期时. Just noticed that. Everything else looks good to me. Thanks.

**Review by litong01 — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/config/analysis/ist0155/index.md (hunk 4 above)
Comment (zirain) [reply content/zh/docs/reference/config/analysis/ist0155/index.md:8]: 应该使用 `升级期间` ？ cc @litong01

Comment (litong01) [reply content/zh/docs/reference/config/analysis/ist0155/index.md:8]: @zirain I feel either 升级期间 or 升级时 should work. Thanks.

Comment (jzhupup) [reply content/zh/docs/reference/config/analysis/ist0155/index.md:8]: ok

**Review by zirain — APPROVED** [review]
lgtm

**Review by litong01 — APPROVED** [review]
Thanks very much!

**Pull request merged**